docs/_sidebar.md

```diff
@@ -12,6 +12,10 @@
   - [require-matching-label](plugins/require-matching-label.md)
   - [hold](plugins/hold.md)
   - [wip](plugins/wip.md)
+  - [size](plugins/size.md)
+  - [assign](plugins/assign.md)
+  - [milestone](plugins/milestone.md)
+  - [needs-rebase](plugins/needs-rebase.md)  
```

Comment:
`needs-rebase` is a external plugin, it would be better to put it together with the other external plugin, like the `require-matching-label` plugin.

Reply:
Or just add an `external` tag after the link?

Reply:
It doesn't seem appropriate. There will be a lot of external plugins, maybe we can add one more directory level, such as:
```
- [插件](plugins.md)
  - [外部插件]
    - [ti-community-owners](plugins/owners.md)
    - [ti-community-lgtm](plugins/lgtm.md)
    - [ti-community-merge](plugins/merge.md)
    - [ti-community-tars](plugins/tars.md)
    - [ti-community-blunderbuss](plugins/blunderbuss.md)
    - [ti-community-label](plugins/label.md)
    - [ti-community-label-blocker](plugins/label-blocker.md)
    - [ti-community-contribution](plugins/contribution.md)
    - [require-matching-label](plugins/require-matching-label.md)
    - [needs-rebase](plugins/needs-rebase.md) 
  - [内置插件]
    - [hold](plugins/hold.md)
    - [wip](plugins/wip.md)
    - [size](plugins/size.md)
    - [assign](plugins/assign.md)
    - [milestone](plugins/milestone.md)
```

Or make the order of the list in `sidebar.md` consistent with the order of the list in `README.md`.